worker/jobTypes/githubJob.js

```diff
@@ -40,7 +42,66 @@ class GitHubJobClass {
         }
         return false;
     }
+    
+    async writeEnvFile(isProdDeployJob){
+      //download published branches file to retrieve prefix and check if repo is versioned 
+      const repoObject = {
+        repoOwner: this.currentJob.payload.repoOwner, repoName: this.currentJob.payload.repoName,
+      };
+      const repoContent = await workerUtils.getRepoPublishedBranches(repoObject)
+      var pathPrefix = repoContent.content.prefix
+
+      if(isProdDeployJob){
+        //versioned repo
+        if(repoContent && repoContent.content.version.active.length > 1){
+          pathPrefix += `/${this.currentJob.payload.branchName}`; 
+        }
+      }
+      // server staging commit jobs
+      else if(this.currentJob.payload.patch && this.currentJob.payload.patchType === 'commit'){
+        const server_user = await this.getUser()
+        pathPrefix += `/${this.currentJob.user}/${this.currentJob.payload.localBranchName}/${server_user}/master`; 
+      }
```

Comment:
can't hardcode the branch here (versioned branches)

Reply:
I believe the branch name can be `{this.currentJob.payload.localBranchName}` for `make stagel commit` jobs as they will be working from the branch they wish to stage